docs/backends/zio.md

```diff
@@ -136,35 +137,30 @@ Please visit [the official documentation](https://armeria.dev/docs/client-factor
 
 ## ZIO environment
 
-As an alternative to effectfully or resourcefully creating backend instances, ZIO environment can be used. In this case, a type alias is provided for the service definition:
+As an alternative to effectfully or resourcefully creating backend instances, ZIO environment can be used. In this case, a type aliases are provided for the service definition:
```

Comment:
Should this be `In this case, type aliases are provided for the service definition`?

Reply:
Maybe `the following type aliases are provided for the service definition` is a better fit.